{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "e2e6524c_c16aa242",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 1000999
      },
      "writtenOn": "2025-08-20T03:34:40Z",
      "side": 1,
      "message": "I did wrong on DataIntegrityChecksum field in the previous patch. The correction is look good.",
      "revId": "a43abba00d50dd0ae7f23704822baf0d65047393",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    }
  ]
}